languages/python/exercises/concept/string-methods/.docs/instructions.md

```diff
@@ -0,0 +1,39 @@
+In this exercise you are helping your younger sister edit her paper for school. The teacher is looking for correct punctuation, grammar, and excellent word choice.
```

Comment:
@ErikSchierboom Do we need an exercise title heading at the top of this file?

Reply:
We don't.